SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -61,19 +61,74 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker")
 	bool bEnableMultiWorker;
 
-public:
 	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
 	TSubclassOf<USpatialMultiWorkerSettings> MultiWorkerSettingsClass;
 
+	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
+	TSubclassOf<USpatialMultiWorkerSettings> EditorMultiWorkerSettingsOverride;
+
+public:
+	/** If command line override for Multi Worker Settings Class is set then use the specified Multi Worker Settings class.
+	 * Else if multi worker is disabled, use the single worker behaviour.
+	 * Else if bForceNonEditorSettings is set use the Multi Worker Settings class.
+	 * Else if the Editor Multi Worker Settings Override is set and we are in the Editor use the Editor Multi Worker Settings.
+	 * Else if the Multi Worker Settings Class is set use it.
+	 * Otherwise default to single worker behaviour.  */
+	TSubclassOf<USpatialMultiWorkerSettings> GetMultiWorkerSettingsClass(bool bForceNonEditorSettings = false) const
+	{
+		const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
```

Comment:
Very nice!
Could we please make a cpp file and add this function there though :)

Reply:
Sure, good point. Created cpp file.